work/worker.go

```diff
@@ -289,11 +295,27 @@ func (self *worker) update() {
 	quitByErr := make(chan bool, 1)
 	go self.handleTxsCh(quitByErr)
 
+	// Initialize reset watchdog
+	if self.nodetype == common.CONSENSUSNODE && self.resetFn != nil && self.resetTimeOut > 0 {
+		callback := func() {
+			logger.Warn("Consensus timeout")
+			self.resetFn()
+		}
+		logger.Info("Initialize auto reset watchdog", "timeout", self.resetTimeOut.String())
```

Comment:
"auto reset" or "auto restart"? It seems that the usages of "reset" and "restart" are mixed. 
"restart" seems to be more appropriate.

Reply:
I changed `reset` to `restart`. Thanks.